addon/-private/execution_context.js

```diff
@@ -1,18 +1,43 @@
 import { getContext } from './helpers';
+import { getContext as getRfc268Context, visit } from './rfc268-helpers';
 import AcceptanceExecutionContext from './execution_context/acceptance';
 import IntegrationExecutionContext from './execution_context/integration';
+import Rfc268Context from './execution_context/rfc268';
 
 const executioncontexts = {
   acceptance: AcceptanceExecutionContext,
-  integration: IntegrationExecutionContext
+  integration: IntegrationExecutionContext,
+  rfc268: Rfc268Context
 };
 
 /*
  * @private
  */
 export function getExecutionContext(pageObjectNode) {
+  // Our `getContext(pageObjectNode)` will return a context only if the test
+  // called `page.setContext(this)`, which is only supposed to happen in
+  // integration tests. `@ember/test-helpers`' `getTestContext()` will return a
+  // context only if the test called one of the RFC268 `ember-qunit`
+  // `setupTest()` methods (e.g. `setupRenderingTest()`,
+  // `setupApplicationTest()`). If neither of these are the case, the only
+  // supported scenario is an acceptance test.
   let testContext = getContext(pageObjectNode);
-  let context = testContext ? 'integration' : 'acceptance';
+  let context;
+  if (testContext) {
+    context = 'integration';
+  } else if (getRfc268Context()) {
+    if (!visit) {
+      throw new Error([
```

Comment:
Under what circumstances could we end up here? When would `getRfc268Context()` return `true` while `visit` is undefined?

Reply:
There were a few minor versions of `@ember/test-helpers` that had `setupRenderingTest()` implemented, but did not have the DOM helpers. In one of my projects, I upgraded `ember-qunit` to start implementing RFC232 rendering tests using `ember-cli-page-object`, but still using `page.setContext()` so it was using the integration execution context.

Since it's in my `yarn.lock`, if I were to update to this version of `ember-cli-page-object` and delete the `page.setContext()`s to try to use the RFC268 execution context, I'd get cryptic `helper is not a function` (or something) errors, so this is meant to clarify what's going on.

There was also a brief window when all the helpers but `visit()` were implemented, but nobody using `ember-cli-page-object` could have actually been using them since there was no execution context that called them. So to keep things clean and simple (ha!), we check for `visit()` which was the last one to be added and guarantees that you can run both RFC232 tests *and* RFC268 tests.